### ISSUE TITLE
Asynchronous subpanels

### ISSUE THREAD

**Issue by Abuelodelanada** [salesagility/SuiteCRM#501]
When having a DetailView with many subpanels the time to load this view may require a lot. This is because the webserver sends the page after rendering it. If a subpanel query takes too many time (Because it is written inefficiently or because the module has millions of records), the whole page takes a lot to be rendered.

A solution for this is having asynchronous subpanels, that loads the data alfter the DetailView is rendered.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Asynchronous subpanels](https://trello.com/c/oULLXzVd/147-asynchronous-subpanels)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
